.github/workflows/e2e-unmanaged-cluster.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: E2E Test - Unamanged Cluster
+
+on:
+  repository_dispatch:
+    types: [daily-build]
+
+  pull_request_target:
+    types: [opened, reopened, synchronize, ready_for_review]
+    paths:
+      - cli/cmd/plugin/unmanaged-cluster
+
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - release-*
+
+    paths:
+      - "cli/cmd/plugin/unmanaged-cluster/*.go"
+      - "cli/cmd/plugin/"
```

Comment:
Do we want to run the unmanaged-cluster e2e tests on any changes in the `plugin` directory? 

For example, this means we will run these e2e tests on changes to the `conformance` for `diagnostics` plugin

Reply:
Oops didn’t noticed thanks.